src/shared-components/Form/Control/Slider.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box } from '@mui/material';
+import Slider from '@mui/material/Slider';
+import { theme } from 'src/styles/theme';
+
+interface CustomSliderProps {
+  defaultValue: number;
+  color: string;
+  thumbbgcolor: string;
+  trackcolor: string;
+  railcolor: string;
+}
+
+export const CustomSlider = ({
```

Comment:
In this component, ask only for one color (base color of the slider. Use that color as color, bg color for thumb and track. For rail, use a `colors.light[20]`

Reply:
Fixed